staging/src/k8s.io/pod-security-admission/webhook/manifests/50-deployment.yaml

```diff
@@ -25,10 +28,11 @@ spec:
             secretName: pod-security-webhook
       containers:
         - name: pod-security-webhook
-          image: k8s.gcr.io/sig-auth/pod-security-webhook:v1.22-alpha.0
+          image: k8s.gcr.io/sig-auth/pod-security-webhook:v1.23-beta.0
           terminationMessagePolicy: FallbackToLogsOnError
           ports:
-            - containerPort: 8443
+            - name: webhook
+              containerPort: 10250
```

Comment:
Why did you change this? I don't have a preference either way, just curious.

Reply:
switched to a named port so the service wouldn't have to have port number knowledge and to make transitioning ports non-disruptive in the future

switched to 10250 since it's a port that is more likely to already be allowed in apiserver/node firewalls to accomodate apiserver -> kubelet traffic

Reply:
Consider adding a comment to that effect?

Reply:
Also, which end does the service port translation happen (apiserver node, or worker node)? If the latter, I imagine the service port would need to be updated too to take advantage of this?

Reply:
> Also, which end does the service port translation happen (apiserver node, or worker node)? If the latter, I imagine the service port would need to be updated too to take advantage of this?

it happens on the apiserver, either via kube-proxy writing iptables rules locally, or via the apiserver following the service -> endpoints mapping

Reply:
> Consider adding a comment to that effect?

done